### PR TITLE
add module BMP280 as subset of BME280

### PR DESCRIPTION
**Pull request by MaBecker**

### CODE REVIEW

**Comment by MaBecker** [comment]
Git and I still have bad moments ....

File: devices/BMP280.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+};
+
+/* Calibration of Pressure, algorithm is taken from the datasheet */
+BMP280.prototype.calibration_P = function(adc_P) {
+/* enable this if you want to be able to debug data values and calibration
+  if (this.debug) {
+    console.log("T1: " + this.dig_T1);
+    console.log("T2: " + this.dig_T2);
+    console.log("T3: " + this.dig_T3);
+    console.log("H1: " + this.dig_H1);
```
Comment (gfwilliams) [inline devices/BMP280.js]: Please could you just remove these 6 lines? I think they reference the values that were removed?

Looks great though - thanks!

Comment (MaBecker) [inline devices/BMP280.js]: thanks for checking - sure - see commit below

**Comment by gfwilliams** [comment]
thanks!

**Comment by MaBecker** [comment]
Which file do I have to update to see BMP280 in the list of modules ?

**Comment by gfwilliams** [comment]
You'll have to wait for me to update the website I'm afraid - it's not automatic.

You should be able to just do `require("https://raw.githubusercontent.com/espruino/EspruinoDocs/master/devices/BMP280.js")` for now though

**Comment by MaBecker** [comment]
ok, I understand, no need to hurry 

require( "with url") works fine 

thank

**Pull request merged**